### DISCUSSION TITLE
[BUG] Plugin will not load the SimpliSafe Login screen in Config UI X

### DISCUSSION THREAD

**Discussion by JTomlin88** [homebridge-simplisafe3/homebridge-simplisafe3#417] [Q&A]
### Description

Trying to setup the plugin and authenticate my SimpliSafe account, but the login screen will not load. 
I've let the screen load for hours and the login part will not come up.
I've uninstalled/reinstalled the plugin multiple times.

### Expected behavior

Launch SimpliSafe Login screen should appear.

### Steps to reproduce

Plugins>Simplisafe3>Settings>Set Up

### Homebridge logs

```bash
N/A, plugin has not been setup.. nothing present in the logs.
```

### Homebridge config.json contents

```json
N/A, plugin has not been setup.. nothing present in the logs.
```

### Screenshots

![image](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/154967466/9eeba99a-8165-46b2-b532-b57c8a273398)
Screen is stuck here and shows the gear turning, but never loads. 

### Plugin version

1.10.13

### Homebridge version

1.7.0

### Node.js version

20.6.1

### Host OS

Windows

**Comment by shamoon** [comment]
Well, nothing has changed and I cant reproduce this (see below). If you are at the setup screen the plugin has been installed, so the logs should show something. Also maybe your browser logs.

What version of config UI X are you running?

<img width="822" alt="Screenshot 2023-12-27 at 12 49 09 AM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/a5aaefb7-f769-41e5-8865-997355496779">

**Reply by JTomlin88** [reply]
Found the log that I was looking for...
![Capture](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/154967466/d9caaf94-fc0e-4898-a59e-1fd955effd28)

**Comment by jaxonashton** [comment]
Getting this same error as well:

<img width="797" alt="image" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/3246415/00a8538b-c8fb-45d9-9242-0449fc638996">

Error from logs is:
<img width="1046" alt="image" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/3246415/f2c9cd99-64d2-4aaf-9e31-9d484e238e80">

Error presents ever time the config screen is loaded. Removing the plugin and re-installing it does not resolve.

Update: Updating HB UI to v1.8.2 did not resolve.
